lib/route.js

```diff
@@ -47,7 +47,7 @@ module.exports = function route (req, res, geojson = {}, options = {}) {
     if (method) {
       result = handleMethodRequest({ method, geojson, req })
     } else if (isRestInfoRequest(route)) {
-      result = restInfo(geojson)
+      result = restInfo(req, geojson)
```

Comment:
can you swap `req` and `geojson` so the signatures look consistent with `serverInfo()` and layerInfo()`

Reply:
Done in [06096f1](https://github.com/koopjs/FeatureServer/pull/230/commits/06096f136c66a9367e3127bc8205d7c8d2af6b35).